corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -19,39 +31,57 @@ hqDefine('accounting/js/pricing_table', function () {
         self.selectCurrentPlan = function () {
             self.selected_edition(self.currentEdition);
         };
+        self.capitalizeString = function (s) {
+            return s.charAt(0).toUpperCase() + s.slice(1);
+        };
+        self.isDowngrade = function (oldPlan, newPlan) {
+            if (oldPlan === 'Enterprise') {
+                if (newPlan === 'Advanced' || newPlan === 'Pro' ||
+                    newPlan === 'Standard' || newPlan === 'Community') {
+                    return true;
+                }
+            }
+            else if (oldPlan === 'Advanced') {
+                if (newPlan === 'Pro' || newPlan === 'Standard' || newPlan === 'Community') {
+                    return true;
+                }
+            } else if (oldPlan === 'Pro') {
+                if (newPlan === 'Standard' || newPlan === 'Community') {
+                    return true;
+                }
+            } else if (oldPlan === 'Standard') {
+                if (newPlan === 'Community') {
+                    return true;
+                }
+            }
+            return false;
+        };
 
         self.form = undefined;
-        self.openDowngradeModal = function(pricingTable, e) {
-            var editionSlugs = _.map(self.editions(), function(e) { return e.slug(); });
+        self.openMinimumSubscriptionModal = function (pricingTable, e) {
             self.form = $(e.currentTarget).closest("form");
-            if (editionSlugs.indexOf(self.selected_edition()) < editionSlugs.indexOf(self.currentEdition)) {
-                var $modal = $("#modal-downgrade");
+
+            var oldPlan = self.capitalizeString(self.currentEdition);
+            var newPlan = self.capitalizeString(self.selected_edition());
+            var newStartDate = self.startDateAfterMinimumSubscription;
+            var mailto = "<a href=\'mailto:billing-support@dimagi.com\'>billing-support@dimagi.com</a>";
+            if (self.isDowngrade(oldPlan, newPlan) && self.subscriptionBelowMinimum) {
+                var $modal = $("#modal-minimum-subscription");
+                $modal.find('.modal-body')[0].innerHTML =
+                    "All CommCare subscriptions require a 30 day minimum commitment. Your current subscription " +
```

Comment:
This should be translated. Since it's a dynamic string, one way to do this is to create an underscore [template](https://underscorejs.org/#template) with the dynamic pieces as variables, then translate that. So something like `_.template(gettext("Your current subscription will be downgraded to <%= newPlan %> on <%= date %>"))({newPlan: newPlan, date: newStartDate})`.

Ideally, the plan names would also have translations; setting them up as constants should help with that.